validation/views.py

```diff
@@ -159,10 +162,16 @@ def advanced_search_results(request):
 
     all_matches.sort(key=lambda phrase: phrase.transcription)
 
-    paginator = Paginator(all_matches, 30)
+    query = f"&transcription={transcription}&translation={translation}&analysis={analysis}&status={status}"
+    for speaker in speakers:
+        query += f"&speaker-options={speaker}"
+
+    query += "&speaker="
```

Comment:
Same comment here: what is the deal with quoting? Is user-input trusted? Is it sanitized before it gets here?

Perhaps consider using [urllib.parse.urlencode](https://docs.python.org/3.7/library/urllib.parse.html#urllib.parse.urlencode)?

```python
>>> from urllib.parse import urlencode
>>> urlencode({"transcrption": "atim", "translation": "dog", "analysis": "atim+N+A+Sg", "speaker": "MAR"})
'transcrption=atim&translation=dog&analysis=atim%2BN%2BA%2BSg&speaker=MAR'
```

(notice what happened to the `analysis`!)

Reply:
Ah! Oh no! I will fix this.